api/types/kustomization.go

```diff
@@ -263,3 +578,21 @@ func (k *Kustomization) Unmarshal(y []byte) error {
 	*k = nk
 	return nil
 }
+
+func stringInSlice(elem string, slice []string) bool {
```

Comment:
api/internal/utils.StringSliceContains

Reply:
Won't be able to use this after moving the code to `kustomize/kustomize/commands/edit/fix` because it's an internal package

Reply:
lol, true